matplotlib2tikz/axes.py

```diff
@@ -389,7 +389,10 @@ def _get_ticks(data, xy, ticks, ticklabels):
         pgfplots_ticks.append(tick)
         # store the label anyway
         label = ticklabel.get_text()
-        pgfplots_ticklabels.append(label)
+        if ticklabel.get_visible():
+            pgfplots_ticklabels.append(label)
+        else:
```

Comment:
By default, pgfplots draws ticklabels. If at least one of them is invisible, I have to precise the entire list of labels I want to draw. Then, it produces something like this (if no ticklabels have to be displayed):
```latex
\nextgroupplot[
xmin=-0.2495, xmax=5.2395,
ymin=-1.1, ymax=1.1,
width=7.5cm,
xtick={-1,0,1,2,3,4,5,6},
xticklabels={},
tick align=outside,
x grid style={white!69.019607843137251!black},
y grid style={white!69.019607843137251!black}
]
```